### ISSUE TITLE
#Issue 10 : Evaluate 43-class model and integrate into Streamlit App

### ISSUE THREAD

**Issue by SajithK203** [SajithK203/Traffic-Sign-Detection#23]
### Summary
Evaluate the performance of the fine-grained 43-class YOLOv8 model on the test split and update the Streamlit demo app to support selecting the 43-class model.
### Tasks
- [ ] Evaluate the fine-tuned 43-class model on the test set:
  ```bash
  venv_gpu\Scripts\python src/evaluate.py --model yolov8 --weights runs/train/gtsdb_yolov8n_43cls_v1/weights/best.pt --data data/processed/gtsdb_fine/test
